{"uuid": "842c861e-241e-449d-bd59-12f12d4df9f5", "children": ["55da08e2-4644-4880-bc1f-9012c34d1481"], "befores": [{"name": "start_appium_desired", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'driver' referenced before assignment\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/zhangxiangkui/Downloads/AutomationTest/testcase/conftest.py\", line 36, in start_appium_desired\n    return driver\n"}, "start": 1587406559062, "stop": 1587406559071}], "start": 1587406559062, "stop": 1587406559122}